src/main/java/org/jenkinsci/plugins/stashNotifier/StashNotifier.java

```diff
@@ -106,7 +106,7 @@
     /**
      * The id of the credentials to use.
      */
-    private String credentialsId;
+    private final String credentialsId;
```

Comment:
shouldn't this break recent setters improvement? with setters the whole configuration should be considered mutable

Reply:
That's the field in `StashNotifier`. There is no setter for it, only a constructor.
The JCasC setters were about the fields in `StashNotifier.DescriptorImpl`.